engine/storage/snapshot/src/org/apache/cloudstack/storage/snapshot/StorageSystemSnapshotStrategy.java

```diff
@@ -440,41 +400,28 @@ private void markAsBackedUp(SnapshotObject snapshotObj) {
 
     @Override
     public StrategyPriority canHandle(Snapshot snapshot, SnapshotOperation op) {
-        long volumeId = snapshot.getVolumeId();
-        VolumeVO volumeVO = _volumeDao.findById(volumeId);
         if (SnapshotOperation.REVERT.equals(op)) {
-            if (volumeVO != null && ImageFormat.QCOW2.equals(volumeVO.getFormat()))
-                return StrategyPriority.DEFAULT;
-            else
-                return StrategyPriority.CANT_HANDLE;
+            return StrategyPriority.CANT_HANDLE;
         }
 
```

Comment:
Good work on rewritting the logic to a slimmer state.

Wouldn't be it more readable if you turn lines 407-422 into a boolean function which would return true if "supportsStorageSystemSnapshots". Snap this chunk of code could prove useful on creating unit tests and documentation as this segment clearly aims to check this condition solely.